### PR TITLE
Add "JSC and dataLog" section

### PR DESCRIPTION
**Pull request by fujii**
Add "JSC and dataLog" section

### CODE REVIEW

File: docs/Build & Debug/Logging.md [hunk 1]
```diff
@@ -108,3 +108,9 @@ Set the `WebCoreLogging` environment variable.
 ## Adding a new log channel
 
 Simply add a line to your framework's `Logging.h` header. Depending on how the accompanying `Logging.cpp` file is set up, you may need to add a parallel line there. That should be all you need. It is acceptable to have log channels in different frameworks with the same name - this is what `LOG_CHANNEL_PREFIX` is for.
+
+## JSC and dataLog
```
Comment (rniwa) [inline docs/Build & Debug/Logging.md:112]: We should probably spell out JavaScriptCore.

File: docs/Build & Debug/Logging.md [hunk 2]
```diff
@@ -108,3 +108,9 @@ Set the `WebCoreLogging` environment variable.
...
+
+WebKit has another logging infrastructure `dataLog`. JSC is mainly using it. To enable the JSC logging, set a environment variable or give a command switch to `jsc`. For example, `JSC_logGC=2` or `run-jsc --logGC=2`. Give `--debug` switch to `run-jsc` script if you build a debug build.
```
Comment (rniwa) [inline docs/Build & Debug/Logging.md:114]: Ditto about spelling out JavaScriptCore.

**Review by rniwa — APPROVED** [review]

File: docs/Build & Debug/Logging.md (hunk 1 above)
Comment (fujii) [reply docs/Build & Debug/Logging.md]: Fixed.

File: docs/Build & Debug/Logging.md (hunk 2 above)
Comment (fujii) [reply docs/Build & Debug/Logging.md]: Fixed.

**Pull request merged**